internal/index_read.go

```diff
@@ -102,26 +103,19 @@ func readBins(r io.Reader, typ string) ([]Bin, *ReferenceStats, error) {
 	return bins, stats, nil
 }
 
-func readChunks(r io.Reader, n int32, typ string) ([]bgzf.Chunk, error) {
+func readChunks(r io.Reader, n int32, typ string, buf [16]byte) ([]bgzf.Chunk, error) {
```

Comment:
I changed it to a slice and get the same performance. I can push that if you prefer.